Makefile

```
@@ -10,18 +10,22 @@ doc/layouts/%.png: doc/%.svg
-	keymap -c keymapper.conf draw -j config/blueclaw.json --keys-only doc/keymap.yaml >doc/keymap.svg
+	uvx --from keymap-drawer keymap -c keymapper.conf draw -j config/blueclaw.json --keys-only doc/keymap.yaml >doc/keymap.svg
 
 doc/keymap_combos.svg: doc/keymap.yaml keymapper.conf config/blueclaw.json
-	keymap -c keymapper.conf draw  --select-layers Base  -j config/blueclaw.json  doc/keymap.yaml >doc/keymap_combos.svg
+	uvx --from keymap-drawer keymap -c keymapper.conf draw  --select-layers Base  -j config/blueclaw.json  doc/keymap.yaml >doc/keymap_combos.svg
 
 doc/keymap.yaml: config/blueclaw.keymap keymapper.conf
-	keymap -c keymapper.conf parse -z config/blueclaw.keymap  >doc/keymap.yaml
+	uvx --from keymap-drawer keymap -c keymapper.conf parse -z config/blueclaw.keymap  >doc/keymap.yaml
+
+cheatsheet: doc/cheatsheet.html
```

checkmake (0.3.2)

[warning] Target "cheatsheet" should be declared PHONY.

(phonydeclared)